### PR TITLE
docs(README): fix urls of README(#522)

### LINKED ISSUE #522: docs: Correct some url errors
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7/CONTRIBUTING.md).

### Description

Wrong url in the README.md
图片

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by error0g**
## Summary about this PR

Issue Number: close #522 

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [x] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: README_zh-hans.md [hunk 1]
```diff
@@ -86,14 +86,14 @@ StoneDB 社区欢迎各种各样的贡献，如为代码库贡献代码，分享
...
+更多信息，可以查看 [Compile StoneDB on Ubuntu 20.04](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu2004/compile-using-ubuntu20.04-for-57/).
 
 ### CentOS 7.x 下编译 StoneDB
 
-更多信息，可以查看 [Compile StoneDB on CentOS 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-centos7).
+更多信息，可以查看 [Compile StoneDB on CentOS 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-centos7/compile-using-centos7-for-57).
 
 ### RedHat 7.x 下编译 StoneDB
-更多信息，可以查看 [Compile StoneDB on RHEL 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-redhat7).
+更多信息，可以查看 [Compile StoneDB on RHEL 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-redhat7/compile-using-redhat7-for-57).
```
Comment (Nliver) [inline README_zh-hans.md]: This should link to the Chinese document~

File: README_zh-hans.md [hunk 2]
```diff
@@ -86,14 +86,14 @@ StoneDB 社区欢迎各种各样的贡献，如为代码库贡献代码，分享
 
 ### Ubuntu 20.04 下编译 StoneDB
 
-更多信息，可以查看 [Compile StoneDB on Ubuntu 20.04](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu20.04).
+更多信息，可以查看 [Compile StoneDB on Ubuntu 20.04](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu2004/compile-using-ubuntu20.04-for-57/).
 
 ### CentOS 7.x 下编译 StoneDB
 
-更多信息，可以查看 [Compile StoneDB on CentOS 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-centos7).
+更多信息，可以查看 [Compile StoneDB on CentOS 7](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-centos7/compile-using-centos7-for-57).
```
Comment (Nliver) [inline README_zh-hans.md]: This should link to the Chinese document~

File: README_zh-hans.md [hunk 3]
```diff
@@ -86,14 +86,14 @@ StoneDB 社区欢迎各种各样的贡献，如为代码库贡献代码，分享
 
 ### Ubuntu 20.04 下编译 StoneDB
 
-更多信息，可以查看 [Compile StoneDB on Ubuntu 20.04](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu20.04).
+更多信息，可以查看 [Compile StoneDB on Ubuntu 20.04](https://stonedb.io/docs/developer-guide/compiling-methods/compile-using-ubuntu2004/compile-using-ubuntu20.04-for-57/).
```
Comment (Nliver) [inline README_zh-hans.md]: This should link to the Chinese document~

**Review by hustjieke — APPROVED** [review]
LGTM

**Comment by hustjieke** [comment]
Nice job!
Thanks for you contribution！
@error0g

**Review by Nliver — APPROVED** [review]
LGTM！
Good job！

**Pull request merged**